Editor/DeploymentScenarios.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+namespace AmazonGameLift.Editor
+{
+    public enum DeploymentScenarios
+    {
+        SingleRegion = 0,
```

Comment:
Please rename to `SingleRegionFleet` so it's clear what this is for. Are the changes for `MultiRegionFleet` in a separate PR? Wondering why that's not an option on here.

Reply:
MultiRegionFleet was not in the mocks. Can you confirm it is a requirement?